src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,31 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+
+    private static final int MAX_SIZE = 5;
+
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        Set<Character> chars = new TreeSet<>();
+
+        try (FileReader fileReader = new FileReader(fileName);
+                BufferedReader bufferedReader = new BufferedReader(fileReader)) {
+            int character = bufferedReader.read();
+            while (character != -1) {
+                if (Character.isLetter(character)) {
+                    chars.add((char) Character.toLowerCase(character));
+                }
+                character = bufferedReader.read();
+            }
+        } catch (IOException exception) {
+            exception.printStackTrace();
+            throw new FileNotFoundException("File " + fileName + " is not found");
+        }
```

Comment:
you are throwing checked Exception from this method anyway and indicate this in method signature. try just removing this catch block and put IOException to method signature